src/algebra/order/floor.lean

```diff
@@ -23,6 +23,7 @@ We define the natural- and integer-valued floor and ceil functions on linearly o
 * `int.floor a`: Greatest integer `z` such that `z ≤ a`.
 * `int.ceil a`: Least integer `z` such that `a ≤ z`.
 * `int.fract a`: Fractional part of `a`, defined as `a - floor a`.
+* `round a`: Nearest integer to `a`. It rounds halves up.
```

Comment:
```suggestion
* `round a`: Nearest integer to `a`. It rounds halves towards infinity.
```
is usually the wording, to avoid confusion over what `-2.5` rounds to.

Reply:
That's a very weird wording to me, but if you say so.